spec/requests/edit_user_spec.rb

```diff
@@ -159,7 +146,6 @@ def sign_in_as_a_valid_user(user)
           "<strong>#{user.reload.unconfirmed_mobile}</strong>."
         )
       expect(flash[:notice]).to eq t('devise.registrations.mobile_update_needs_confirmation')
-      expect(user.reload.mobile).to be_nil
```

Comment:
I think we want to keep these expectations (several of them have been deleted from this file), but instead of checking that the mobile is nil, we need to check that it hasn't changed. The point of this expectation is to make sure that the mobile doesn't change until the new one is confirmed.